lib/plain/entities/concept-scheme.ts

```diff
@@ -0,0 +1,114 @@
+import { OptionalDefaults } from '../wrappers/wrap'
+import {
+  CursorPaginatedCollectionProp,
+  DeleteConceptSchemeParams,
+  GetConceptSchemeParams,
+  GetManyConceptSchemeParams,
+  GetOrganizationParams,
+  UpdateConceptSchemeParams,
+} from '../../common-types'
+import { ConceptSchemeProps, CreateConceptSchemeProps } from '../../entities/concept-scheme'
+import { Patch } from 'json-patch'
+
+export type ConceptSchemePlainClientAPI = {
+  /**
+   * Create Concept Scheme
+   * @param payload the Concept Scheme
+   * @returns the created Concept Scheme
+   * @throws if the request fails
+   * @see {@link https://www.contentful.com/developers/docs/references/content-management-api/#/reference/taxonomy/create-a-concept-scheme}
+   * @example
+   * ```javascript
+   * const concept = await client.conceptScheme.create({
+   *   organizationId: '<organization_id>',
+   * }, conceptSchemeProps);
+   * ```
+   */
+  create(
+    params: OptionalDefaults<GetOrganizationParams>,
+    payload: CreateConceptSchemeProps
+  ): Promise<ConceptSchemeProps>
+
+  /**
+   * Update Concept Scheme
+   * @param params Concept Scheme ID and Concept Scheme version
+   * @param payload Concept Scheme patch
+   * @returns the updated Concept Scheme
+   * @throws if the request fails
+   * @see {@link https://www.contentful.com/developers/docs/references/content-management-api/#/reference/taxonomy/concept-scheme}
+   * @example
+   * ```javascript
+   * const updatedConcept = await client.conceptScheme.update({
+   *   organizationId: '<organization_id>',
+   * }, conceptSchemePatch);
+   * ```
+   */
+  update(
+    params: OptionalDefaults<UpdateConceptSchemeParams>,
+    payload: Patch
+  ): Promise<ConceptSchemeProps>
+
+  /**
+   * Get Concept Scheme
+   * @param params Concept Scheme ID
+   * @returns the Concept Scheme
+   * @throws if the request fails
+   * @see {@link https://www.contentful.com/developers/docs/references/content-management-api/#/reference/taxonomy/concept-scheme}
+   * @example
+   * ```javascript
+   * const concept = await client.conceptScheme.get({
+   *   organizationId: '<organization_id>',
+   *   conceptSchemeId: '<concept_scheme_id>',
+   * });
+   * ```
+   */
+  get(params: OptionalDefaults<GetConceptSchemeParams>): Promise<ConceptSchemeProps>
```

Comment:
`OptionalDefaults` resolves to `Record<string, unknown>`, which allows to pass any object

Reply:
@maxcheremisin good catch - we've solved it for the new endpoints introduced in this PR, but something we should look into in general (like fixing the actual util type `OptionalDefaults`